about-pses/example-pse.md

```diff
@@ -1,90 +1,372 @@
 
-Answer the following prompts:
+## Example Input/Output
 
-1. List five or more questions whose answers would clarify the problem statement
-2. List two sets of example arguments and the expected return value for these arguments
-3. Divide the project prompt into at least two different sub-problems
-4. Pick one of those problems from #3 and write a numbered list explaining how to solve that problem without using full lines of code
-5. Write the pseudocode or code solution for your answer for #4
+| Input (Argument of the function) | Expected Output (Return value of the function) | Explanation|
+|--|--|--|
+|player_1 = `"rock"` <br> player_2 = `"scissors"`| `"Player 1 wins!"`|`"rock"` beats `"scissors"`|
+|player_1 = `"rock"` <br> player_2 = `"rock"`| `"It's a tie!"`|`player_1` and `player_2` played the same move|
 
-## Example Answers
+### Complete Rock, Paper, Scissors Ruleset
+| input (first argument)| input (second argument) | output |
+|--|--|--|
+|**player_1** |**player_2**|**return value**|
+|`"rock"`|`"rock"`|`"It's a tie!"`|
+|`"paper"`|`"paper"`|`"It's a tie!"`|
+|`"scissors"`|`"scissors"`|`"It's a tie!"`|
+|`"rock"`|`"paper"`|`"Player 2 wins!"`|
+|`"rock"`|`"scissors"`|`"Player 1 wins!"`|
+|`"paper"`|`"rock"`|`"Player 1 wins!"`|
+|`"paper"`|`"scissors"`|`"Player 2 wins!"`|
+|`"scissors"`|`"rock"`|`"Player 2 wins!"`|
+|`"scissors"`|`"paper"`|`"Player 1 wins!"`| 
 
-### #1 Five or More Clarifying Questions
+## Prompts
 
-1.  Should I return the restaurant's name, or the dictionary that represents the restaurant data, or the rating?
-1.  What is the name of the key in each dictionary whose value is the rating?
-1.  What kind of data-type is rating stored in?
-1.  What should happen if there's more than one highest-rated restaurant?
-1.  Is the restaurant's name included in this dictionary?
-1.  What else is included in each dictionary?
-1.  Will there ever be zero restaurants? What should happen?
-1.  What should I do if there is no rating?
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: paragraph
+* id: 164a62ea-f936-4c6a-8b74-6d8248ba9c9d
+* title: Ask Clarifying Questions
+* topics: pse
+##### !question
 
-### #2 Two Example Inputs/Outputs
+List five or more questions whose answers would clarify the problem statement
 
-1.  Example one
-    - Input: `[{name: "Grillby's", rating: 1}, {name: "Crow's Nest", rating: 5}]`
-    - Output: `{name: "Crow's Nest", rating: 5}`
-1.  Example two
-    - Input: `[{name: "Crow's Nest", rating: 1}]`
-    - Output: `{name: "Crow's Nest", rating: 1}`
+##### !end-question
 
-### #3 Two Sub-Problems
+##### !explanation 
 
-1.  How do you check/iterate over all of the restaurants?
-1.  How do you compare the ratings between restaurants?
+Here are some example clarifying questions:
 
-### #4 Steps to Solve a Sub-Problem
+1. How should the function handle invalid user input (i.e. player_1 = ["lizards"](https://bigbangtheory.fandom.com/wiki/Rock,_Paper,_Scissors,_Lizard,_Spock))?
```

Comment:
I think adding the link is fun but a bit distracting. Would recommend to keep it without and the people who know the reference will still get a laugh :) 

```suggestion
1. How should the function handle invalid user input (i.e. player_1 = "lizards")?
```